### ISSUE TITLE
M600 based filament change treatment now missing post change extrusion from version 2.2.0

### ISSUE THREAD

**Issue by p17blo** [UnchartedBull/OctoDash#2186]
Since upgrading octodash to V2.2.0, activating the filament change, with config options set to M600, used to allow for a post change extrusion of any length as deemed by the operator. This amount was entered into the touch screen at the point immediately AFTER loading the new filament. 

Since upgrading to v2.2.0, immediately after loading the new filament, the screen momentarily shows some kind of progress bar and instead of displaying a screen and asking for user input as to how much filament should be extruded before the hot end is cooled, it now just flashes and immediately starts to cool down. 

Raspberry Pi 3B+
octoprint 1.6.1
Octodash 2.2.0
7” touchscreen

**Comment by UnchartedBull** [comment]
Yes this screen was removed from the M600 option since M600 has a purge option on it's own. The general gist was that people, that are using M600 want as little intervention from OctoDash as possible, thus this screen was removed.

Please correct me if I'm wrong though.
